### PR TITLE
Updated-afl

### PR DESCRIPTION
**Pull request by PankajKataria**
@radare ping
This is with reference to #4607

### CODE REVIEW

File: libr/core/anal.c [hunk 1]
```diff
@@ -1406,6 +1406,14 @@ R_API int r_core_anal_fcn_list(RCore *core, const char *input, int rad) {
 	} else if (rad == 'j')  {
 		r_cons_printf ("[");
 	}
+	if(rad == 'o'){
```
Comment (radare) [inline libr/core/anal.c]: missing space before parenthesis

**Comment by radare** [comment]
I think that this output must be different:

* as long as function name is the only field with variable size, you should put it as the las column.
* aligning columns like this doesnt works when using > 32bit addresses
* Having a header is nice when the list is short, but you can't count lines with ~?

Lot of tests are not passing because of those changes, let's review them and add comments to get better output. thanks

```
-0x00401000  59    1     fcn.00401000
-0x0040103b  15    3     fcn.0040103b
-0x0040104a  109   4     fcn.0040104a
-0x004010c7  29    1     fcn.004010c7
-0x004010e4  148   4     fcn.004010e4
-0x00401335  262   3     loc.00401335
-0x00404ad7  8     1     fcn.00404ad7
+Offset     Size  nbbs  Function Name                  Min_Bound  Max_Bound  Calls Vars Args Xref Framesize
+======     ====  ====  =============                  =========  =========  ===== ==== ==== ==== =========
+0x00401000 59    1     fcn.00401000                   0x00401000 0x00401000 3     2    0    6    59  
+0x0040103b 15    3     fcn.0040103b                   0x0040103b 0x00401045 2     4    2    39   15  
+0x0040104a 109   4     fcn.0040104a                   0x0040104a 0x004010b1 8     2    1    13   109 
+0x004010c7 29    1     fcn.004010c7                   0x004010c7 0x004010c7 1     0    1    3    29  
+0x004010e4 148   4     fcn.004010e4                   0x004010e4 0x00401172 13    2    2    21   148 
+0x00401335 262   3     loc.00401335                   0x00401335 0x00401427 2     4    2    37   262 
+0x00404ad7 8     1     fcn.00404ad7                   0x00404ad7 0x00404ad7 0     0    0    0    8   
```

**Comment by radare** [comment]
ping

**Comment by jduck** [comment]
I like the idea of having column names in general but I agree that it complicates (and more importantly pollutes) things when using row/column selectors etc. Perhaps in the future there could be a way to show the header in such a way that it doesn't. Something like "stderr" vs "stdout"... It seems like a big change though and would likely take some time to build something like that out.

PS. Having column names in the code also helps in self-documentation ;-)

**Comment by radare** [comment]
Ideally columnation will be done with an rcons filter like we do with ~ for grepping.

Headers are a problem for prsing so if there are many columns the meaning of them is confusing and forces to add some hints like cc=6 which makes it wider

Theres also aflj which lists in json which is better for parsing. But not for wc-l or grepping

**Comment by radare** [comment]
This is related https://github.com/radare/radare2/issues/4621

**Pull request closed without merging**